TLSharp.Core/TelegramClient.cs

```diff
@@ -313,17 +324,23 @@ public async Task SendPingAsync()
             await _sender.SendPingAsync();
         }
 
-        public async Task<TLAbsMessages> GetHistoryAsync(TLAbsInputPeer peer, int offset, int max_id, int limit)
+        /// <summary>
+        /// messages.getHistory#dcbb8260 peer:InputPeer offset_id:int offset_date:date add_offset:int limit:int max_id:int min_id:int hash:int = messages.Messages
```

Comment:
this XML doc is a bit strange, either make it human readable or remove it please

Reply:
FYI: SearchUserAsync also has the XML doc summary.